libraries/esp8266/examples/SigmaDeltaDemo/SigmaDeltaDemo.ino

```diff
@@ -0,0 +1,84 @@
+/*
+  Sigma Delta Generator example
+  
+  This example demonstrates the use of the ESP8266 internal hardware sigma delta source.
+  Each GPIO pin or multiple pins can be connected to the one sigma delta source.
+  THE sigma delta output frequency is defined by the ESP8266 hardware as:
+
+  FREQ = 80,000,000/prescaler * target /256  HZ,          0<target<128
+  FREQ = 80,000,000/prescaler * (256-target) /256  HZ,    128<target<256
```

Comment:
Are these constant even in 2x mode where CPU_F = 160,000,000?

Reply:
Good point, i never tested 160MHz. Will verify and correct accordingly!

Reply:
Checked : CPU frequency has no influence on the SD frequencies